tests/std/tests/P0896R4_ranges_alg_unique/test.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+using P = pair<int, int>;
+
+// Validate dangling story
+STATIC_ASSERT(same_as<decltype(ranges::unique(borrowed<false>{})), ranges::dangling>);
+STATIC_ASSERT(same_as<decltype(ranges::unique(borrowed<true>{})), ranges::subrange<int*>>);
+
+struct instantiator {
+    static constexpr P expected[4] = {{0, 99}, {1, 47}, {3, 99}, {4, 47}};
+
+    static constexpr auto make_counter(size_t& count) {
+        return [&count](const int a, const int b) {
+            ++count;

```

Comment:
As this test includes `<algorithm>` and uses `std`, I believe that we should avoid shadowing `count`.
```suggestion
    static constexpr auto make_counter(size_t& counter) {
        return [&counter](const int a, const int b) {
            ++counter;
```